test_unittests_server.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import server
+import pytest
+
+
+################
+# UNIT TESTS
+################
+
+
+def test_response_ok():
+    response = server.response_ok(b"123456789", "text/plain")
+    assert "text/plain" in response
+    assert '46' in response
```

Comment:
this would be a fantastic place to really deconstruct the response and verify that it is a proper HTTP response.  You are not doing so in this case.